File: app/src/main/java/com/leaguechampions/data/model/RiotRealm.java
package com.leaguechampions.data.model;

import com.google.gson.annotations.SerializedName;

public class RiotRealm {

    @SerializedName("n")
    private RiotVersion version;

    public String getVersion() {
        return version.champion;
    }

    private static class RiotVersion {
        String champion;
    }
}

File: libraries/core/src/main/java/com/leaguechampions/libraries/core/data/model/RiotRealm.kt
package com.leaguechampions.libraries.core.data.model

import com.google.gson.annotations.SerializedName

class RiotRealm(
    @SerializedName("n")
    private val version: RiotVersion
) {

    fun getVersion(): String {
        return version.champion
    }

    class RiotVersion {
        internal lateinit var champion: String
    }
}
